DesignSafe-CI/portal
# DES-773 - Add message to notify duplicate email in Edit Profile

## Overview: ##
Added a way to handle the duplicate email error in the interface.

## PR Status: ##


## Related Jira tickets: ##

* [DES-773](https://jira.tacc.utexas.edu/browse/DES-773)

## Summary of Changes: ##
Changed `apps/accounts/views.py` to add an error on duplicate emails rather than a silent 500 server error when editing profile.

## Testing Steps: ##
1.Go to the "Edit Profile" page and enter an existing email that is not associated with the currently logged-in account.
2. Confirm that the error message occurs.

## UI Photos:
![Screenshot from 2020-10-23 12-39-48](https://i.imgur.com/YXwjwh7.png)



## Notes: ##
I was not sure how to test this given that I was using my other account when testing it for myself.
